### ISSUE TITLE
Deep-G-Prop shouldn't really have the "Prop" part?

### ISSUE THREAD

**Issue by JJ** [JJ/2021-cec-deep-g-prop#23]
I'm not 100% sure, but if it's using Stochastic Gradient Descent, is that the same as backpropagation?

**Comment by lulivi** [comment]
I think The backpropagation algorithm used is SGD, but it is still backpropagation because it learns from the errors, right?

**Comment by JJ** [comment]
It's not propagated back, I'm afraid... Let's say it's got it for historical reasons.

**Comment by pacastillo** [comment]
I agree. We could keep the old name, or define a new one later, maybe in a next paper.
Maybe something that includes "gradient"?

**Comment by JJ** [comment]
We might want to try other algorithms that do not necessarily include
"gradient" in their name. There are like a dozen algorithms in scikit-learn
that we might want to try later.
